web_src/less/themes/theme-auto.less

```diff
@@ -0,0 +1,3 @@
+@media (prefers-color-scheme: dark) {
```

Comment:
Are you sure this is actually right? Doesn't this just not include any CSS for light themes?

Reply:
Correct.
That assumes that the light theme only sets color values, and nothing else.
If it sets other values, than we will have hard to trace-back bugs.

Reply:
Right, what I'm saying is that all CSS is conditional on the dark theme, not just the colours, i.e. it falls back to nothing on light theme. Unless the light theme is somehow also required by the theme.